frontend/lib/content-transformer-plugin.js

```diff
@@ -0,0 +1,134 @@
+const fs = require("fs");
```

Comment:
This is the Webpack plugin I've been hacking on to generate static content from collections of files. It could use some documentation - but until then, the modules under `frontend/lib/content` are working examples of use.

This is a little over-engineered because it supports async/await all the way down. But, it handles standalone content generation in scripts. It also works as a Webpack plugin that sets up file watchers for all the source files to re-generate static content as needed.